src/cpp/src/image_generation/flux_pipeline.hpp

```diff
@@ -109,23 +109,8 @@ namespace genai {
 
 class FluxPipeline : public DiffusionPipeline {
 public:
-    explicit FluxPipeline(PipelineType pipeline_type) : DiffusionPipeline(pipeline_type) {
-        // TODO: support GPU as well
-        const std::string device = "CPU";
-
-        if (m_pipeline_type == PipelineType::IMAGE_2_IMAGE || m_pipeline_type == PipelineType::INPAINTING) {
-            const bool do_normalize = true, do_binarize = false, gray_scale_source = false;
-            m_image_processor = std::make_shared<ImageProcessor>(device, do_normalize, do_binarize, gray_scale_source);
-            m_image_resizer = std::make_shared<ImageResizer>(device, ov::element::u8, "NHWC", ov::op::v11::Interpolate::InterpolateMode::BICUBIC_PILLOW);
-        }
-
-        if (m_pipeline_type == PipelineType::INPAINTING) {
-            bool do_normalize = false, do_binarize = true;
-            m_mask_processor_rgb = std::make_shared<ImageProcessor>(device, do_normalize, do_binarize, false);
-            m_mask_processor_gray = std::make_shared<ImageProcessor>(device, do_normalize, do_binarize, true);
-            m_mask_resizer = std::make_shared<ImageResizer>(device, ov::element::f32, "NCHW", ov::op::v11::Interpolate::InterpolateMode::NEAREST);
-        }
-    }
+    explicit FluxPipeline(PipelineType pipeline_type) :
```

Comment:
we can drop such constructor as it does not do anything useful.

Places like ` : FluxPipeline(pipeline_type)` will be replaced with ` : DiffusionPipeline(pipeline_type)`